src/brainseg/create_fh_seg.py

```diff
@@ -0,0 +1,121 @@
+import os
+import sys
+import subprocess
+import nibabel as nib
+import numpy as np
+import scipy.ndimage
+import shutil
+
+def run_synthseg(conda_env, input_nifti, output_nifti, synthseg_script, synthseg_dir):
+    python_bin = os.path.join(conda_env, "bin", "python")
+    env = os.environ.copy()
+    env["PYTHONPATH"] = synthseg_dir
```

Comment:
I'm not sure setting the `PYTHONPATH` is enough to make the complete environment work. Specifically, SynthSeg relies on CUDA libraries and would be surprised if running it on the GPU works.